src/main/java/RestApi/movies/model/Comments.java

```diff
@@ -4,7 +4,7 @@
 
     private String user;
     private String message;
-    private int dateCreated;
+    private String dateCreated;
```

Comment:
Could we find some way to turn this into a `LocalDate`? It's currently in seconds since epoch